nativelib/src/main/resources/scala-native/gc/shared/Safepoint.c

```diff
@@ -12,6 +12,9 @@
 #include <windows.h>
 #else // Unix
 #include <sys/mman.h>
+#ifndef MAP_NORESERVE
```

Comment:
I suggest making this change FreeBSD specific.
That is, restrict the scope of variant handling to only
what we know & have exercised. 

Let future OSen add their own variant handling, if necessary.
< #if defined(__FreeBSD__)
< #if !defined(MAP_NORESERVE)
< #define MAP_NORESERVE 0 // concept does not exist on this FreeBSD; clear bit.
< #endif
< #endif